_infra/helm/locust/locustfiles/main.py

```diff
@@ -125,7 +125,7 @@ def reformat_date(date):
 
 # Collection exercise loading
 def load_collection_exercises(auth):
-    config = json.load(open("/mnt/locust/collection-exercise-config.json"))
+    config = json.load(open(".//collection-exercise-config.json"))
```

Comment:
The existing /mnt/ file location is specific to the containerised helm deployment. The change here is to allow the file to be references from a local non-containerised workspace.